root/layout/components/sidebar/SidebarAcousticBrainz.js

```diff
@@ -0,0 +1,81 @@
+/*
+ * @flow
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import hydrate from '../../../utility/hydrate';
+
+import {SidebarProperty, SidebarProperties} from './SidebarProperties';
+
+const SidebarAcousticBrainz = ({recording}: {recording: RecordingT}) => {
+  const [count, setCount] = React.useState(0);
+  const [isCountLoaded, setIsCountLoaded] = React.useState(false);
+  const [data, setData] = React.useState({});
+  const [isDataLoaded, setIsDataLoaded] = React.useState(false);
```

Comment:
Might make sense to just make the default value for `data` `null` - then you can just check `data != null` to determine if it's loaded rather than relying on `isDataLoaded`.

Reply:
Good point, fixed